client/containers/DeleteEventContainer.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { DeleteEventForm } from './index'
+import { ModalWithForm } from '../components'
+import { connect } from 'react-redux'
+import * as actions from '../actions'
+import * as selectors from '../selectors'
+
+/**
+* Modal for adding an agenda
+*/
+export const DeleteEvent = ({ show, handleHide, eventDetail }) => (
+    <ModalWithForm
```

Comment:
Why do we need a `<ModalWithForm/>` ? I don't see any field to fill in this modal.
It seems more to be a simple modal to me